apps/web/package.json

```diff
@@ -8,7 +8,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "copy:pdfjs": "node ../../scripts/copy-pdfjs.cjs"
+    "copy:pdfjs": "node ../../scripts/copy-pdfjs.cjs",
+    "lint:fix": "next lint --fix"
```

Comment:
Toss this one next to the other linting task. I prefer to keep them grouped by purpose where possible 😄

Reply:
Done ✅